src/transformers/pipelines/token_classification.py

```diff
@@ -192,7 +192,6 @@ def preprocess(self, sentence, offset_mapping=None):
         truncation = True if self.tokenizer.model_max_length and self.tokenizer.model_max_length > 0 else False
         model_inputs = self.tokenizer(
             sentence,
-            return_attention_mask=False,
```

Comment:
`return_attention_mask=True` is also incorrect because FNet doesn't expect an attention mask.

Reply:
Maybe FNet will continue to exhibit the flaw that pad tokens modify the output, I don't know enough about it though.

Reply:
You will thus get the attention mask since you don't remove it afterward, but I'm guessing that's the whole point?

Reply:
Actually it seems the FNet tokenizer doesn't return attention mask if we don't ask  for it. (Which is fair since the model doesn't seem to accept them).